docs/source/entities.mdx

```diff
@@ -55,14 +55,14 @@ type Product @key(fields: "id") {
 
 The `@key` directive defines the entity's **primary key**, which consists of one or more of the type's `fields`. In the example above, the `Product` entity's primary key is its `id` field.
 
-**Every instance of an entity must be uniquely identifiable by its `@key` fields.** This is what enables your router to associate field data from _different_ subgraphs with the _same_ entity instance.
```

Comment:
I guess it must be true. (Otherwise, how would entities work?)

I'm a little confused though—if the keys can be mismatched, how do the keys help the router associate entity fields across subgraphs?

Reply:
The other message describes it better (I think at least)- it's a weird thing.